fastlane_core/lib/fastlane_core/tool_collector.rb

```diff
@@ -67,5 +119,32 @@ def did_show_message?
       File.write(path, '1') unless did_show
       did_show
     end
+
+    def determine_version(name)
+      begin
+        # We need to pre-load the version file because tools that are invoked through their actions
+        # will not yet have run their action, and thus will not yet have loaded the file which defines
+        # the module and constant we need.
+        require File.join(name.to_s, "version")
+
+        # Go from :credentials_manager to 'CredentialsManager'
+        class_name = name.to_s.fastlane_class
```

Comment:
Here is a similar [workaround](https://github.com/fastlane/fastlane/pull/4575/files)